Source/Shared/BPConfiguration.m

```diff
@@ -609,6 +609,9 @@ - (BOOL)parseXcSchemeFile:(NSString *)schemePath withError:(NSError *__autorelea
 
 - (BOOL)validateConfigWithError:(NSError *__autoreleasing *)errPtr {
     BOOL isdir;
+
+    [BPUtils enableDebugOutput:self.verboseLogging];
```

Comment:
we don't want this pushed?

Reply:
I don't see why not. Without it the `-v` options is essentially not wired. I realize it's lots and lots of debug output, but it's behind the `-v` flag so it should be ok, no?

Reply:
fair